bundesliga_site/urls.py

```diff
@@ -28,5 +28,7 @@
     url(r'^accounts/logout/$', logout, name='logout'),
     url(r'^password_reset/$', login, name='password_reset'),
     url(r'^$', HomeView.as_view(template_name='index.html'), name='index'),
+    url(r'^select_events/$', SelectEvents.as_view(template_name='organizer/select_events.html'), name='select_events'),
```

Comment:
I believe the `template_name='organizer/select_events.html'` is not necessary since it's already declared in the `SelectEvents` class.